packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -42,22 +45,28 @@ class FlatTable<T> extends StatefulWidget {
 
   final ItemCallback<T> onItemSelected;
 
+  final ColumnData<T> startingSortColumn;
```

Comment:
what does the word starting mean here? Can this just be sortColumn or can it just be a column index?